### DISCUSSION TITLE
글 작성시 줄바꿈 표현

### DISCUSSION THREAD

**Discussion by lamPolar** [RollingPaper42/Front-End#26] [Q&A]
조회 페이지 디자인이 지금 작성자가 의도한대로 줄바꿈을 보여주지 않는데, 글을 작성하는 순간부터 줄바꿈을 아예 금지시켜야 될까요?
사용자가 글을 쓴 뒤에 그 부분을 알게되면 사용성 측면에서 엥? 하는 부분이 생길 것 같은데,  
@Arkingco @lyssoi @dokoh1 어떻게 생각하세요?

**Comment by Arkingco** [comment]
피그마에서도 영어는 단어 하나를 전부 보존하기 위해 줄바꿈 시켜 버리네요
한글은 줄바꿈 시키지 않아서 피그마로 봤을 땐 괜찮게 보이는데
한글 자체는 text align justify 옵션으로 양쪽 정렬이 가능해서 피그마 디자인 처럼 만들 수 있으니 한글 우선으로 작업하고 영어는 무시하고 작업 합시다
<br>
![image](https://github.com/RollingPaper42/Front-End/assets/28525747/5c176b3c-155f-4d3a-9fba-695378454255)

**Comment by lyssoi** [comment]
혹시 개행('\n')을 금지한다는 말씀이시라면 매우 좋다고 생각합니다.
